### ISSUE TITLE
Please enable this package on js file (JSX)

### ISSUE THREAD

**Issue by yudium** [sh4nks/auto_close#4]
JSX is like html so this package may available in *.js or *.jsx file

**Comment by yudium** [comment]
Now it works. Just copy key binding from this repo and change the operand value.

```
    {
    	"keys": [">"], "command": "auto_close_tag", "args": {"prefix": ">"}, "context":
        [
            {"key": "selector", "operator": "equal", "operand": "source.js meta.group.braces.round.js", "match_all": true},
            {"key": "preceding_text", "operator": "not_regex_contains", "operand": "(?:\\<(?:img|br|hr|meta|link|base|input)\\>[^<]*)|/$"}
        ]
    }
```

I am new user in sublime. I am happy now after understand how it works.

**Comment by sh4nks** [comment]
Sorry for my late reply. I have fixed this issue in the linked commit. I will release a new version as soon as I have fixed #1 as well. 

Thanks for your code snippet @yudisupriyadi123!
